shopware/platform
# feat: Admin vite plugins

# Description
Adds Vite plugin support to the Shopware Admin build setup, allowing plugins to leverage the new build system while maintaining existing functionality.

## Key Changes
- Implemented `plugins.vite.ts` script to handle plugin builds and dev mode
- Added new Vite plugins:
  - `shopware-vite-plugin-vue-globals` for Vue instance management
  - `shopware-vite-plugin-override-component` for automatic override registration

## Test Instructions
1. Enable the feature flag:
ADMIN_VITE=1
2. Run `composer build:js:admin` with any plugin active without custom webpack config.
2.1. I tested with SwagCustomizedProducts and Commercial. (**Commercial needs changes made for the build to succeed**)
4. Build should succeed
